MessageBird/Objects/VoiceCalls/VoiceCallFlow.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace MessageBird.Objects.VoiceCalls
+{
+
+    public class Options
+    {
+
+        [JsonProperty("destination")] public string Destination { get; set; }
```

Comment:
Can you reformat this file so the JsonProperty attribute is above the property? 

[JsonProperty("destination")]
public string Destination { get; set; }